index.html

```diff
@@ -67,7 +67,7 @@
       <div class="col-xs-12 col-sm-8 col-md-7">
       <h1>Phinx</h1>
       <h2>PHP Database Migrations For Everyone</h2>
-      <p><a type="button" class="btn btn-primary" href="#installModal" role="button" data-toggle="modal" onclick="_gaq.push(['_trackEvent', 'Hero Unit actions', 'Install', 'Install v0.10.8']);">INSTALL NOW v0.10.8</a>
+      <p><a type="button" class="btn btn-primary" href="#installModal" role="button" data-toggle="modal" onclick="_gaq.push(['_trackEvent', 'Hero Unit actions', 'Install', 'Install v0.11.3']);">INSTALL NOW v0.11.3</a>
```

Comment:
Should we only mention the current major? Avoids manual fix of this file all the time.

Reply:
Personally, I'd say create a bash script that takes as parameter the version to set that uses regex to update these two spots. This then could be run on tagged releases on Travis and use Travis to push the change to GH on build.

Reply:
For now we dont have any such thing, until then I would prefer we keep human error to a minimum.

Reply:
Fair enough, remove bugfix version.